packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -909,8 +910,8 @@ You very likely have the wrong "binaryTarget" defined in the schema.prisma file.
       headers['X-transaction-id'] = interactiveTransaction.id
     }
 
-    this.currentRequestPromise = this.connection.post('/', stringifyQuery(query.query), headers)
-    this.lastQuery = query.query
+    this.currentRequestPromise = this.connection.post('/', JSON.stringify(query), headers)
+    // this.lastQuery = query.query
```

Comment:
Why is this not needed anymore?  I never took the time to understand its purpose (explanations welcome :D)

Reply:
Oops, i think it was not intentional, let me see.

Reply:
Restored that part. As far as I can tell, it is used only in case of panic for appending `Engine Query` section to automatic github issue.  At the same time, i think, sanitiation function is buggy have been sanitising it down to `{ 
"X": {} } for as long as it existed (2 years, according to git blame). So, maybe we don't need this part at all?

Reply:
Optimally we would have a test that caught this...